_data/componentlist.json

```diff
@@ -164,10 +164,10 @@
             "url": "/documentation/header",
             "text": "Header & Subnav",
             "type": "component",
-            "ux": 10,
-            "ui": 10,
+            "ux": 20,
+            "ui": 20,
```

Comment:
I think this is still under implementation. Search and the Header Actions section are still pending, no?